File: logica-general/ejercicio-027/resoluciones/lucas-pajarito/playlist.py
playlist = [
    {
        "titulo": "Blinding Lights",
        "artista": "The Weeknd",
        "duracion": 200
    },
    {
        "titulo": "Shape of You",
        "artista": "",
        "duracion": 233
    },
    {
        "titulo": "Believer",
        "artista": "Imagine Dragons",
        "duracion": 0
    },
    {
        "titulo": "Levitating",
        "artista": "Dua Lipa",
        "duracion": 203
    },
    {
        "titulo": "",
        "artista": "Coldplay",
        "duracion": 245
    }
]


def detectar_inconsistencias(playlist):

    for cancion in playlist:

        if cancion["titulo"] == "":
            print("Error: La canción no tiene un título registrado.")

        elif cancion["artista"] == "":
            print(f'{cancion["titulo"]}: No tiene un artista registrado.')

        elif cancion["duracion"] <= 0:
            print(f'{cancion["titulo"]}: La duración es inválida.')

        else:
            print(f'{cancion["titulo"]}: Información correcta.')


detectar_inconsistencias(playlist)

File: logica-general/ejercicio-007/resoluciones/lucas-pajarito/playlist.js
const playlist = [
    {
        titulo: "Blinding Lights",
        artista: "The Weeknd",
        duracion: 200
    },
    {
        titulo: "Shape of You",
        artista: "Ed Sheeran",
        duracion: 233
    },
    {
        titulo: "Bad Habits",
        artista: "Ed Sheeran",
        duracion: 0
    },
    {
        titulo: "Levitating",
        artista: "Dua Lipa",
        duracion: 203
    },
    {
        titulo: "As It Was",
        artista: "Harry Styles",
        duracion: -180
    }
];

function revisarPlaylist(playlist) {

    playlist.forEach(cancion => {

        if (cancion.duracion <= 0) {
            console.log(`${cancion.titulo}: Duración inválida. Revisar la información de la canción.`);
        } else {
            console.log(`${cancion.titulo}: Información correcta.`);
        }

    });

}

revisarPlaylist(playlist);